docs/application/vstools/tools/project-wizard.md

```diff
@@ -31,10 +31,12 @@ When you create a new project with a specific template, the Project Wizard autom
 To create a Tizen .NET application project:
 
 1.  To open the Project Wizard:
+
     a.  In the Visual Studio menu, go to **File &gt; New &gt; Project &gt; Templates &gt; Visual C\# &gt; Tizen**.
+
     b.  Select a template for your project.
 
-        The Project Wizard creates the project based on the selected template.
+    The Project Wizard creates the project based on the selected template.
```

Comment:
Please indent 3 spaces to align with 2nd ordered list.

Reply:
Done.